assets/js/modules/ads/components/setup/SetupMainPAX.js

```diff
@@ -0,0 +1,233 @@
+				/* eslint-disable sitekit/acronym-case */
+				// Disabling rule because function and property names
+				// are expected in current format by PAX API.
+				const customerData = await app
+					.getServices()
+					?.accountService?.getAccountId( {} );
+				const conversionTrackingData = await app
+					.getServices()
+					?.conversionTrackingIdService?.getConversionTrackingId(
+						{}
+					);
+
+				if (
+					customerData.externalCustomerId &&
+					conversionTrackingData.conversionTrackingId
+				) {
+					setExtCustomerID( customerData.externalCustomerId );
+					setPaxConversionID(
+						conversionTrackingData.conversionTrackingId
+					);
+					/* eslint-enable sitekit/acronym-case */
+
+					const { error } = await submitChanges();
+
+					// Since callback is mostly invoked pretty early, site info is not resolved yet.
+					// We need to ensure data is resolved otherwise admin url will be undefined and finishSetup
+					// will trigger console error instead of redirecting to the dashboard.
+					await registry
+						.__experimentalResolveSelect( CORE_SITE )
+						.getSiteInfo();
+
+					const adminURL = registry
+						.select( CORE_SITE )
+						.getAdminURL( 'googlesitekit-dashboard', {
+							notification: 'authentication_success',
+							slug: 'ads',
+						} );
+
+					if ( ! error ) {
+						finishSetup( adminURL );
+					}
+				}
+			}
+		},
+		[
+			setExtCustomerID,
+			setPaxConversionID,
+			submitChanges,
+			registry,
+			finishSetup,
+		]
+	);
+
+	const clickCallback = useCallback( () => {
+		if ( ! hasAdwordsScope ) {
+			navigateTo( OAuthURL );
+		} else {
+			setPax( 1 );
+		}
+	}, [ navigateTo, setPax, hasAdwordsScope, OAuthURL ] );
+
+	return (
+		<div className="googlesitekit-setup-module googlesitekit-setup-module--ads">
+			<div className="googlesitekit-setup-module__step">
+				<div className="googlesitekit-setup-module__logo">
+					<AdsIcon width="33" height="33" />
+				</div>
+
+				<h2 className="googlesitekit-heading-3 googlesitekit-setup-module__title">
+					{ _x( 'Ads', 'Service name', 'google-site-kit' ) }
+				</h2>
+			</div>
+
+			<AdBlockerWarning />
+
+			{ ! isAdBlockerActive &&
+				( pax && hasAdwordsScope ? (
+					<PAXEmbeddedApp displayMode="setup" onLaunch={ onLaunch } />
+				) : (
+					<Fragment>
+						<div className="googlesitekit-setup-module__description">
+							{ createInterpolateElement(
+								__(
+									'Add your conversion ID below. Site Kit will place it on your site so you can track the performance of your Google Ads campaigns. <a>Learn more</a>',
+									'google-site-kit'
+								),
+								{
+									a: (
+										<SupportLink
+											path="/google-ads/thread/108976144/where-i-can-find-google-conversion-id-begins-with-aw"
+											external
+										/>
+									),
+								}
+							) }
+							<br />
+							{ __(
+								'You can always change this later in Site Kit Settings.',
+								'google-site-kit'
+							) }
```

Comment:
Paragraphs come with default margin, which creates spacing between text. I will open a new issue for changing this, as also new styling needs to be applied for this paragraphs, in order not to have spacing etc, scope of this issue is quite big, and since this was not part of it, better to focus effort on this separately

Reply:
Right, but what's wrong with the spacing between the text? I think it's fine to render these as separate paragraphs. 😄

Reply:
Oh, I thought there _wasn't_ a Figma design.

Okay, that's fine for now.